ui/components/src/model/theme.ts

```diff
@@ -20,6 +20,17 @@ export interface PersesChartsTheme {
     width: number;
     color: string;
   };
+  /**
+   * Theming for the container that wraps a chart.
+   */
+  container: {
```

Comment:
Does this general structure/naming of the theme entry look good to you? Very open to feedback now since changing it later can cause breaking changes.

Reply:
I just took a peek at EChartsTheme's JSON to see what they do: https://echarts.apache.org/en/theme-builder.html

The only thing coming to mind right now is maybe we want to call this `containerPadding` instead of `container.padding`

In ECharts, it seems like most of the top-level fields in "theme" are the individual panels, which makes sense to me... each panel has its own options

But then there are some global fields, like `color` and `backgroundColor`

It does seem nice to have a default padding size for all panels, but I can't think of anything else we'll put inside `container`.

Thus, collapsing the fields `container` and `padding` into just `containerPadding`

But I'm not 100% of this idea, just something to think about. I can also see that reasons to have `container` mean that if we do come up with another property of container that we want global styles for, we can put it here. 

Just not sure if global styles for all panels will be common

Reply:
I was also struggling with that. I opted for the object to leave room to grow without requiring breaking changes, but it is a little awkward as empty as it is right now, and it's hard to predict what we'll need in there in the future. @sjcobb -- you have any strong feels one way or the other to sway us?

Reply:
This is all outside the `echartsTheme` so we can really name it whatever we want, `sparkline` for instance is also a custom property that I made up.  Normally with ECharts you customize the padding using the [grid properties](https://echarts.apache.org/en/option.html#grid), but since we've built our own Legend and YAxisLabel components with React, that only helps us for the inner portions rendered to canvas. 

I'm good with `container.padding` though! Maybe we'll think of other properties to add inside container later like you said (other apps might want to change the border or something)